Makefile

```diff
@@ -190,12 +192,15 @@ META.coq: META.coq.in
 
 .PHONY: clean cleankeepvo objclean cruftclean indepclean docclean archclean optclean clean-ide mlgclean depclean cleanconfig distclean voclean timingclean alienclean
 
-clean: objclean cruftclean depclean docclean camldevfilesclean
+clean: objclean cruftclean depclean docclean camldevfilesclean gramlibclean
 
 cleankeepvo: indepclean clean-ide optclean cruftclean depclean docclean
 
 objclean: archclean indepclean
 
+gramlibclean:
```

Comment:
Should be PHONY
We should split the big PHONY for clean targets and put each next to the corresponding target ala
~~~makefile
.PHONY: clean
clean: objclean ...

.PHONY: cleankeepvo
cleankeepvo: indepclean ...

...
~~~

Reply:
Fixed.